src/SeleniumLibrary/keywords/browsermanagement.py

```diff
@@ -117,6 +117,70 @@ def open_browser(self, url, browser='firefox', alias=None,
         uses. Notice that prior to SeleniumLibrary 3.0, the library
         contained its own profile that was used by default.
 
+        Optional ``options`` argument allows to define browser specific
+        Selenium options. Example for Chrome, the ``options`` argument
+        allows defining the following
+        [https://seleniumhq.github.io/selenium/docs/api/py/webdriver_chrome/selenium.webdriver.chrome.options.html#selenium.webdriver.chrome.options.Options|methods and attributes]
+        and for Firefox these
+        [https://seleniumhq.github.io/selenium/docs/api/py/webdriver_firefox/selenium.webdriver.firefox.options.html?highlight=firefox#selenium.webdriver.firefox.options.Options|methods and attributes]
+        are available. Please note that not all browsers supported by the
+        SeleniumLibrary have Selenium options available, please consult
+        the Selenium documentation which browser do support the Selenium
+        options. Selenium options are also supported, when ``remote_url``
+        argument is used.
+
+        The SeleniumLibrary ``options`` argument accepts Selenium
+        options in three different formats 1) As a sting, 2) List containing
+        dictionaries 3) Python object which is an instance of the
+        Selenium options object.
+
+        The sting format is allows to define Selenium options methods
+        or attributes and it's arguments in string format. Methods and
+        attributes are separated by a comma and arguments are separated
+        with a colon. Example like this:
+        `argument:arg1,other_argument:arg2:arg3,attribute:value`.
+
+        The method and attributes are case and space sensitive
+        and must match to the Selenium options methods and attributes names.
+        It is possible to have space before and after the comma for readability.
+        Example `argument:arg1,argument:arg2` and
+        `argument:arg1 , argument:arg2` are converted to same
+        object internally. Spaces around arguments are not removed.
```

Comment:
Does "Spaces around arguments are not removed." mean that in the space in the previous example `argument:arg1 ,` would be significant?

Reply:
Also, `argument:arg1` is somewhat confusing because it's using word argument twice.

Reply:
1) Yes it would be meaningful and therefore it's a bad example. Will fix it. After reading that whole part, perhaps it would be better just to say that: `space after comma is allowed for readability.` Because that is the way it works. Also should check that parsing is aligned with the plugin API parsing because they do similar thing. Perhaps even see can both parsing live in same code.

2) Very true, something like `method:arg1` would be better.